airflow/__init__.py

```diff
@@ -79,3 +79,16 @@ class AirflowViewPlugin(BaseView):
 class AirflowMacroPlugin(object):
     def __init__(self, namespace):
         self.namespace = namespace
+
+
+from airflow import operators  # noqa: E402
+from airflow import sensors  # noqa: E402
+from airflow import hooks  # noqa: E402
+from airflow import executors  # noqa: E402
+from airflow import macros  # noqa: E402
+
+operators._integrate_plugins()
+sensors._integrate_plugins()  # noqa: E402
```

Comment:
This one shouldn't need a noqa should it?

Reply:
I just follow the original code.
https://github.com/apache/incubator-airflow/pull/3906/files